var/spack/repos/builtin/packages/apptainer/package.py

```diff
@@ -64,3 +70,25 @@ def flag_handler(self, name, flags):
         # Certain go modules this build pulls in cannot be built with anything
         # other than -O0. Best to just discard any injected flags.
         return (None, flags, None)
+
+    # They started vendoring the fuse bits and assume they'll be in the
+    # libexec/apptainer prefix as a result. When singularity is run with
+    # suid it doesn't search the user's $PATH for security reasons.
+    # Since we don't use the vendored deps and instead install them in
+    # their own prefixes they are not found by default.
+    # This is likely only relevant for 1.3:, but it should be fine everywhere
+    @run_after("install")
+    def fix_binary_path(self):
+        paths = []
+        # default value
+        binpath = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
This adds `/usr/local` which is where environment views may be placed. Here the assumption is that they are system paths. I don't know if this matters in this particular case... but it may cause spack packages to be found in the $PATH that just happen to be installed in the same environment.

Reply:
Do we actually want to look into system directories at all?

Reply:
That's the default out of box value for the search paths in apptainer. It's possible there are other utilities that apptainer's going to want from the system beyond the ones it nominally wants in it's libexec prefix.

Reply:
alternatively we could instead add symlnks into prefix.libexec.apptainer for the four utilities instead of modifying the apptainer config. I actually like that a bit more.